src/shims.h

```diff
@@ -31,9 +31,12 @@
 #include <pthread.h>
 #else // defined(_WIN32)
 #include "shims/generic_win_stubs.h"
-#include "shims/generic_sys_queue.h"
 #endif // defined(_WIN32)
 
+#if defined(_WIN32) || defined(__ANDROID__)
```

Comment:
I'm tempted to add Linux to this clause as well ...

Reply:
Makes sense, let me know if you want me add it to this PR.

Reply:
A separate PR is probably better.  Lets get android working again!